### PR TITLE
Update maintainers.json

### PR DESCRIPTION
**Pull request by nathanchere**
Joining v3 maintainers

### CODE REVIEW

**Comment by neenjaw** [comment]
Hey @nathanchere, I'm happy to merge, I just want to make sure that you intended to mark `"show_on_website": true` where as @ihid's general msg in the #v3 channel suggested to mark it as false.

If he (or another) said to mark it as true, just let me know and I will merge. 

```
New maintainers:

- In order to get into the correct teams, and get the correct permissions set, we need you to PR your details to the config/maintainers.json file in the track repository (The file is at https://github.com/exercism/$SLUG/blob/master/config/maintainers.json). Please set show_on_website: false for now, and we'll flip this in a couple of weeks for you :slightly_smiling_face:
- You should also PR yourself to the "Contributing Maintainers" part of the v3 repo's language specific maintainers.md file. This file is used purely as an easy way of finding the other maintainers. File at https://github.com/exercism/v3/blob/master/languages/$SLUG/maintainers.md).

Old maintainers

- You should also check you're in the track config/maintainers.json. We've already had four maintainers who were missing and lost permissions in the last few hours and we had to rest.
- You should add yourself as a "Senior Maintainer" to the maintainers.md file
```

**Comment by neenjaw** [comment]
Also, on a side note for @cohencarlisle or @devonestes, right now when I am looking at this it is at 50 minutes of Travis CI time because of the version testing matrix and also that dialyzer isn't cached. 

Is there a way to cache some part of the dialyzer to help speed up this build? Or would that require a move to GitHub Actions?

**Comment by ErikSchierboom** [comment]
@neenjaw I don't see @nathanchere in the current [maintainers.json](https://github.com/exercism/elixir/blob/master/config/maintainers.json), so I think `show_on_website` should indeed be set to `false`.

File: config/maintainers.json [hunk 1]
```diff
@@ -50,6 +50,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": "Exercism has been a great way for me to stay current and expand my developer experience in new areas.  I am excited to be working with the elixir track, because I appreciate the clarity/idiomaticity of the elixir language.  I want others to enjoy this track as much as I have."
+    },
+    {
+      "github_username": "nathanchere",
+      "alumnus": false,
+      "show_on_website": true,
```
Comment (neenjaw) [inline config/maintainers.json]:
```suggestion
      "show_on_website": false,
```

**Comment by NobbZ** [comment]
> Is there a way to cache some part of the dialyzer to help speed up this build?

Yes

> Or would that require a move to GitHub Actions?

I have not looked into it, but given the structure exercism exercises usually have, dialyzer doesn't give anything. So I'd probably just remove it…

**Comment by Cohen-Carlisle** [comment]
I'm pretty sure that dialyzer run just errors out but since it isn't set to ci mode it still exits with status code 0. It can probably be removed entirely.

**Pull request merged**